docs/source/performance.md

```diff
@@ -2,6 +2,16 @@
 
 Scylla Operator 1.6 introduces a new experimental feature allowing users to optimize Kubernetes nodes.
 
+:::{warning}
+The tool performing the tuning doesn't support reverting the changes (see [scylladb/seastar#2350](https://github.com/scylladb/seastar/issues/2350)).
+
+The only complete way to revert them to disable the tuning **and** restart all the nodes hosting your cluster.
+
+There also were cases of performance degradation after applying the tuning (f.e. [scylladb/scylla-operator#2017](https://github.com/scylladb/scylla-operator/issues/2017)).
```

Comment:
referencing one *open* issue that has not yet been even triaged, doesn't fit a general claim made here (`were case`)